torchao/experimental/kernels/mps/src/dispatch.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+// All rights reserved.
+//
+// This source code is licensed under the license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+#include <functional>
+
+namespace torchao {
+namespace kernels {
+namespace mps {
+namespace lowbit {
+namespace dispatch {
+
+inline void dispatch_mm(
+    id<MTLComputeCommandEncoder> encoder,
+    int32_t maxThreadsPerGroup,
+    int32_t M,
+    int32_t N,
+    int32_t K) {
+  (void)K;
```

Comment:
Or you can use good old `__attribute__((unused))` if you want to stick to C++14 
```suggestion
    [[maybe_unused]]  int32_t K) {
```